### PR TITLE
Similiar Stripe jobs info box  

### PR DESCRIPTION
**Pull request by wandererfrog**
Creates highlight links that map to information boxes.
When the link is hovered the content changes.

Import the component
```
import InteractiveBox from '../components/InteractiveBox/InteractiveBox'
```

Link the information boxes to the links.

``` 
<InteractiveBox>
              <Article link="link_1" logo="information.svg" title="Slide 1" />
              <Article link="link_2" logo="information.svg" title="Slide 2" />
  </InteractiveBox>
```

Add the links on the content.
```
<P>
              Når vi derimod forurener med drivhusgasser, som forårsager klodens
              globale opvarmning, så er det helt cool og ganske gratis. Forestil
              dig to virksomheder der begge sælger T-shirts.<a className="highlight-link" id="link_1" href="#">Den første fabrik
              er drevet på elektricitet fra vindmøller.</a> Den anden på el fra
              olie. Den oliedrevne fabrik forurener langt mere end den
              vindmølledrevne. <a id="link_2" href="#" className="highlight-link">Alligevel er der ingen, der kommer efter dem med
              den klækkelige bøde, som du fik, da du kastede skraldeposen ud af
              vinduet.</a>
</P>
```

Take care for the usability aspect of it, do not overflow with content.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1326,6 +1321,11 @@ alphanum-sort@^1.0.0:
   resolved "https://registry.yarnpkg.com/alphanum-sort/-/alphanum-sort-1.0.2.tgz#97a1119649b211ad33691d9f9f486a8ec9fbe0a3"
   integrity sha1-l6ERlkmyEa0zaR2fn0hqjsn74KM=
 
+animate.css@^3.7.2:
```
Comment (Jacse) [inline yarn.lock]: 1) Is this a new dependency? It's not in `package.json`

**Comment by Jacse** [comment]
Ref #53

**Comment by Jacse** [comment]
We should also consider how these should behave on small screens. Maybe they should just not be shown?

**Comment by wandererfrog** [comment]
Oops, forgot to remove the animate.css from the yarn.lock =)

Stripe shows a different version of it on mobile,  popup fixed in the bottom of the screen.

File: yarn.lock (hunk 1 above)
Comment (wandererfrog) [reply yarn.lock]: Removed.

**Comment by wandererfrog** [comment]
This is going to fail because the links were removed from the DOM. I created a new PR with a fix.

**Pull request merged**